mount.go

```diff
@@ -257,7 +258,19 @@ func mountStorage(storage pb.Storage) error {
 		return err
 	}
 
-	return mount(storage.Source, storage.MountPoint, storage.Fstype, flags, options)
+	var fsType = storage.Fstype
```

Comment:
Please just set `storage.Fstype` to the proper fstype as detected in kata-runtime. Then the agent doesn't need to specially find out the fstype.

Reply:
But i think that the `storage.Fstype` was set `bind` and we need to pass it to the `libcontainer` pkg, And if we want mount the blockdevice as a dir, we must specify the real filesystem such as `ext4`,`xfs`, etc.  So i temporarily discard the bind option here and find out the fstype again.